tests/rados/test_bluestoretool_workflows.py

```diff
@@ -0,0 +1,272 @@
+    Returns:
+        1 -> Fail, 0 -> Pass
+    *** Currently, covers commands/workflows valid only for collocated OSDs
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    rhbuild = config.get("rhbuild")
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    bluestore_obj = BluestoreToolWorkflows(node=cephadm)
+
+    out, _ = cephadm.shell(args=["ceph osd ls"])
+    osd_list = out.strip().split("\n")
+
+    try:
+        # Execute ceph-bluestore-tool --help
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Running cbt help for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.help(osd_id=osd_id)
+        log.info(out)
+
+        # Execute ceph-bluestore-tool fsck --path <osd_path>
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Running consistency check for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.run_consistency_check(osd_id=osd_id)
+        log.info(out)
+        assert "success" in out
+
+        # Execute ceph-bluestore-tool fsck --deep --path <osd_path>
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Running deep consistency check for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.run_consistency_check(osd_id=osd_id, deep=True)
+        log.info(out)
+        assert "success" in out
+
+        if rhbuild.startswith("6"):
+            # Execute ceph-bluestore-tool qfsck --path <osd_path>
+            osd_id = random.randrange(len(osd_list))
+            log.info(
+                f"\n --------------------"
+                f"\n Running quick consistency check for OSD {osd_id}"
+                f"\n --------------------"
+            )
+            out = bluestore_obj.run_quick_consistency_check(osd_id=osd_id)
+            log.info(out)
+            assert "success" in out
+
+            # Execute ceph-bluestore-tool allocmap --path <osd_path>
+            osd_id = random.randrange(len(osd_list))
+            log.info(
+                f"\n --------------------"
+                f"\n Fetching allocmap for OSD {osd_id}"
+                f"\n ---------------------"
+            )
+            out = bluestore_obj.fetch_allocmap(osd_id=osd_id)
+            log.info(out)
+            assert "success" in out
+
+        # Execute ceph-bluestore-tool repair --path <osd_path>
+        osd_id = random.randrange(len(osd_list))
+        log.info(
+            f"\n --------------------"
+            f"\n Run BlueFS repair for OSD {osd_id}"
+            f"\n --------------------"
+        )
+        out = bluestore_obj.repair(osd_id=osd_id)
+        log.info(out)
+        assert "success" in out
+
+        """
```

Comment:
Please add some details about tasks to be done further. like a ` # tbd: file bug, or complete changes with next PR ` etc.

Reply:
Added details about next steps and reason for exclusion